src/config_directives.c

```diff
@@ -164,16 +164,36 @@ CFGFUN(for_window, const char *command) {
     TAILQ_INSERT_TAIL(&assignments, assignment, assignments);
 }
 
-void create_gaps_assignment(const char *workspace, bool inner, const long value) {
+void create_gaps_assignment(const char *workspace, const char *scope, gaps_t gaps) {
     DLOG("Setting gaps for workspace %s", workspace);
 
     struct Workspace_Assignment *assignment;
     TAILQ_FOREACH(assignment, &ws_assignments, ws_assignments) {
         if (strcasecmp(assignment->name, workspace) == 0) {
-            if (inner)
-                assignment->gaps.inner = value;
-            else
-                assignment->gaps.outer = value;
+            if (!strcmp(scope, "inner"))
```

Comment:
I know this wasn't from you, originally, but could we please also add `{}` here?